docs/documentation/admin/workflows/admin_workflow_groups.md

```diff
@@ -0,0 +1,264 @@
+---
+id: admin_workflow_groups
+title: I. Workflow Groups
+sidebar_label: I. Workflow Groups
+---
+import useBaseUrl from '@docusaurus/useBaseUrl'; 
+import Highlight from '@theme/Highlight';
+
+<img alt="design" className="img_sizing" src={useBaseUrl('img/design/Workflows.svg')} />
+<br/>
+
+<div className="alert alert--secondary">
+
+## Summary
```

Comment:
missing i18n

Reply:
Oh! Thanks!

Just in case, there's a command that can automatically create the header IDs. Just in case we missed some, I'll run it again before beginning to translate.